static/app/views/starfish/modules/databaseModule/panel.tsx

```diff
@@ -68,63 +76,45 @@ export default function QueryDetail({
 function QueryDetailBody({row}: EndpointDetailBodyProps) {
   const theme = useTheme();
   const location = useLocation();
-
-  const TABLE_QUERY = `
-    SELECT transaction, count() AS count, quantile(0.75)(exclusive_time) as p75
-    FROM spans_experimental_starfish
-    WHERE module = 'db'
-    AND group_id = '${row.group_id}'
-    GROUP BY transaction
-    ORDER BY count DESC
-    LIMIT 10
+  const pageFilter = usePageFilters();
+  const {startTime, endTime} = getDateFilters(pageFilter);
+  const DATE_FILTERS = `
+    start_timestamp > fromUnixTimestamp(${startTime.unix()}) and
+    start_timestamp < fromUnixTimestamp(${endTime.unix()})
   `;
 
-  const GRAPH_QUERY = `SELECT
-      toStartOfInterval(start_timestamp, INTERVAL 12 HOUR) as interval,
-      quantile(0.75)(exclusive_time) as p75,
-      count() as count
-      FROM spans_experimental_starfish
-      WHERE module = 'db'
-      AND group_id = '${row.group_id}'
-      GROUP BY interval
-      ORDER BY interval asc
-   `;
-
-  const EVENT_COUNT_QUERY = `
-  SELECT transaction, count(DISTINCT transaction_id) as uniqueEvents
-    FROM spans_experimental_starfish
-    WHERE transaction
-      IN (SELECT transaction FROM spans_experimental_starfish WHERE module='db' AND group_id='${row.group_id}')
```

Comment:
I removed this subquery, I'm pretty sure its the same query if you just move the conditions back to the top level?